modules/indexing/src/test/java/org/apache/ignite/internal/processors/query/h2/twostep/GridMapQueryExecutorTest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.sqltests;
+
+import java.util.Arrays;
+import java.util.List;
+import org.apache.ignite.Ignite;
+import org.apache.ignite.internal.IgniteEx;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+
+public class GridMapQueryExecutorTest extends BaseSqlTest {
+    
```

Comment:
There is no test for this code to check the logs:
            // Log the error along with the SQL query and parameters.
            if (qry != null) {
                U.error(log, "Failed to execute local query: " + qry.query() + " with parameters: " + Arrays.toString(qry.parameters(params)) + ". Node ID: " + node.id(), e);
            } else {
                U.error(log, "Failed to execute local query and query details are not available", e);
            }
            
            The test has to verify that the appropriate  "Failed to execute local query: " is inside the log

Reply:
Implemented tests to verify log entries are as expected under error conditions, using a mocked logger for controlled test outcomes.